server/graphql/v2/interface/Account.js

```diff
@@ -756,9 +761,9 @@ export const AccountFields = {
   paymentMethods: {
     type: new GraphQLNonNull(new GraphQLList(PaymentMethod)),
     args: {
-      // TODO: Should filter by providerType
       types: { type: new GraphQLList(GraphQLString), deprecationReason: '2020-07-26: Please use type (singular)' },
       type: { type: new GraphQLList(GraphQLString) },
+      service: { type: PaymentMethodService },
```

Comment:
It's expected to be a list right?

Reply:
@znarf : Thanks. My mistake. Corrected it. 👍🏽